index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE HTML>
+<HTML>
+  <head>
+    <title>
+Microverse trial by Sahir and Lauriane
+    </title>
+    <nav id="navbar">
+     <header>
+       <a href="#Introduction" class="nav-link">Introduction</a>
+       <a href="#Chapter_1" class="nav-link">Chapter 1</a>
+       <a href="#Chapter_2" class="nav-link">Chapter 2</a>
+       <a href="#Chapter_3" class="nav-link">Chapter 3</a>
+       <a href="#Chapter_4" class="nav-link">Chapter 4</a>
+       </header> 
+    </nav>
+  </head>
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <style>
+    #navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0px;
+  left: 0px;
+  width: 300px;
+  height: 100%;
+  border-right: solid;
+  border-color: rgba(0, 22, 22, 0.4);
+}
+    @media (max-width: 75em) {
+      html {
+        font-size: 60%;
+      }
+    }
+    
+    
+    @media (max-width: 61.25em) {
+      html {
+        font-size: 58%;
+      }
+    }
+    
+    
+    @media (max-width: 28.75em) {
+      html {
+        font-size: 55%;
+      }
+    </style>
+<main id="main-doc">
+<section class="main-section" id="Introduction">
+<header>Introduction
+  </header>
+  <p>
+    </p>
+  <p>
+    </p>
+    <code>
+      </code>
+   <ul>
+     <li>
+       </li>
+    </ul>
+</section>
+<section class="main-section" id="Chapter_1">
+<header>Chapter 1
+  </header>
+  <p>
+    </p>
+  <p>
+    </p>
+    <code>
+      </code>
+      <ul>
+     <li>
+       </li>
+    </ul>    
+</section>
+<section class="main-section" id="Chapter_2">
+ <header>Chapter 2
+  </header>
+  <p>
+    </p>
+  <p>
+    </p>
+    <code>
+      </code>
+      <ul>
+     <li>
+       </li>
+    </ul> 
+</section>
+<section class="main-section" id="Chapter_3">
+<header>Chapter 3
+  </header>
+  <p>
+    </p>
+  <p>
+    </p>
+    <code>
+      </code>
+      <ul>
+     <li>
+       </li>
+    </ul>  
+</section>
+<section class="main-section" id="Chapter_4">
+<header>Chapter 4
+  </header>
+  <p>
+    </p>
+  <p>
+    </p>
+    <code>
+      </code>
+      <ul>
+     <li>
+       </li>
+    </ul>  
+</section>
```

Comment:
- [ ] I think you can try to replace the header tag wrapping the text "Chapter#" by an appropriate tag for title e.g: h1, h2, etc 😎😎
- [ ] you can try also to add some text inside your p elements so your document can be more descriptive😎